Plugins/Public/playercntl_plugin/Message.cpp

```diff
@@ -1527,17 +1527,6 @@ namespace Message
 			uint iClientSystemID = 0;
 			pub::Player::GetSystem(iClientID, iClientSystemID);
 
-			bool timeSent = false;
-
-			if (mapInfo[iClientID].bShowChatTime)
-			{
-				// Send time with gray color (BEBEBE) in small text (90) above the chat line.
-				bSendingTime = true;
-				HkFMsg(iClientID, L"<TRA data=\"0xBEBEBE90\" mask=\"-1\"/><TEXT>" + XMLText(GetTimeString(set_bLocalTime)) + L"</TEXT>");
-				bSendingTime = false;
-				timeSent = true;
-			}
-
```

Comment:
Removed - reason why the time was sent to everyone, no matter what 'diemsg' settings they had > made all the if-statements at the bottom useless, thank to 'timeSent = true;'